lib/internal/event_target.js

```diff
@@ -305,7 +305,7 @@ class Event {
     if (!isEvent(this))
       throw new ERR_INVALID_THIS('Event');
     if (value) {
-      this.stopPropagation();
+      this.#propagationStopped = true;
```

Comment:
Is setting `cancelBubble` supposed to be one-way? The proposed change doesn't modify `this.#propagationStopped` when `value` is `false`